packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -193,6 +193,80 @@ void main() {
     skip: kIsWeb, // [intended]
   );
 
+  // Regression test for https://github.com/flutter/flutter/issues/127597
+  testWidgets(
+    'TapRegionSurface detects outside right click',
+        (WidgetTester tester) async {
+      String tappedOutside = '';
+      final GlobalKey keyA = GlobalKey();
+      final GlobalKey keyB = GlobalKey();
+      final GlobalKey keyC = GlobalKey();
+      await tester.pumpWidget(
+        MaterialApp(
+          home: Align(
+            alignment: Alignment.topLeft,
+            child: Column(
+              children: <Widget>[
+                const Text('Outside'),
+                Material(
+                  child: TextFormField(
```

Comment:
This test should be for `EditableText` specifically. You can add a different test in the Material library that passes the  `groupId` to a `TextField` and one for a `TextFormField`. In those tests you then use a finder to grab the EditableText and check the group id.

Reply:
I will modify the unit test